### PR TITLE
Update cp210x.c added easywave Eldat usb stick 

### PR DESCRIPTION
**Pull request by nattlip**
added Eldat easywave usb stick Rx09

### CODE REVIEW

**Comment by pelwell** [comment]
cp210x.c is an unmodified upstream source file, so you need to get the Linux devs to approve your change (which should be straightforward).

Johan Hovold <johan@kernel.org> seems to be the main maintainer of that file: 
```
$ scripts/get_maintainer.pl drivers/usb/serial/cp210x.c
Johan Hovold <johan@kernel.org> (maintainer:USB SERIAL SUBSYSTEM)
Greg Kroah-Hartman <gregkh@linuxfoundation.org> (supporter:USB SUBSYSTEM)
linux-usb@vger.kernel.org (open list:USB SERIAL SUBSYSTEM)
linux-kernel@vger.kernel.org (open list)
```
Use `git format-patch` to create a patch file or `git send-mail` to email it directly. https://github.com/nattlip/linux/blob/rpi-4.9.y/Documentation/SubmittingPatches explains the process, but a polite request to Johan may be sufficient.

File: drivers/usb/serial/cp210x.c [hunk 1]
```diff
@@ -152,6 +152,19 @@ static const struct usb_device_id id_table[] = {
 	{ USB_DEVICE(0x12B8, 0xEC62) }, /* Link G4+ ECU */
 	{ USB_DEVICE(0x13AD, 0x9999) }, /* Baltech card reader */
 	{ USB_DEVICE(0x1555, 0x0004) }, /* Owen AC4 USB-RS485 Converter */
+	{ USB_DEVICE(0x155A, 0x1005) }, /* ELDAT Easywave Transceiver */
+        { USB_DEVICE(0x155A, 0x1006) }, /* ELDAT USB Transceiver Easywave */
```
Comment (tracernz) [inline drivers/usb/serial/cp210x.c:156]: You will also need to replace spaces with a tab here. ;)

**Comment by 6by9** [comment]
https://www.mail-archive.com/linux-usb@vger.kernel.org/msg100209.html adds the ELDAT Easywave RX09 (155A:1006). I'm guessing that'll be merged in 4.17 as it was only sent earlier this month.
It is cc'ed to stable though, so it may come back to 4.14 automagically.

**Comment by JamesH65** [comment]
EasyWave RX09 now in 4.19/5.0. Not sure about the rest of the items from the PR, but this still need to be done upstream rather than here.

**Comment by 6by9** [comment]
Also in 4.14 as it was cc'ed to stable - https://github.com/raspberrypi/linux/commit/c0aed3667b3d55a8c49d4411627a41d1fbfb73e0#diff-3f072f93107fdabaa29b0c87481685a7

**Pull request closed without merging**